README.md

```diff
@@ -506,7 +506,7 @@ The following list is a complete list of features that can be opt in and out.
 
 | Name    |  Description | Additional Dependency |Default?|
 |---------|--------------|--------|------------|
-| datetime | Eanble the date and time coordinate support | chrono | Yes |
```

Comment:
This readme is automatically generated with doc-template/readme.template.md. If you don't mind could you please change it there and run the doc-template/update-readme.sh please. Thanks!